pandas/tests/reshape/test_pivot.py

```diff
@@ -1625,3 +1625,9 @@ def test_isleapyear_deprecate(self):
 
         with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
             assert isleapyear(2004)
+
+    def test_issue_13292(self):
+        # The below shouldn't raise an exception anymore.
+        frame = pd.DataFrame({'foo': [1, 2, 3]})
+        pd.pivot_table(frame, index=['foo'], aggfunc=len, margins=True,
+                       margins_name=u'\u0394\u03bf\u03ba\u03b9\u03bc\u03ae')
```

Comment:
get the ``result = pd.pivot_table(....)``

and compare to a constructed ``expected`` with ``tm.assert_frame_equal``

Reply:
@jreback Could you suggest what the constructed `expected` should look like? I'm a bit new and unsure of what should go there. Here's what I've got so far:

```python
    def test_pivot_margins_name_unicode(self):
        # issue #13292
        greek = u'\u0394\u03bf\u03ba\u03b9\u03bc\u03ae'
        frame = pd.DataFrame({'foo': [1, 2, 3]})
        table = pd.pivot_table(frame, index=['foo'], aggfunc=len, margins=True,
                               margins_name=greek)
        expected = pd.DataFrame({}, columns=['foo'], index=[1, 2, 3, greek])
        tm.assert_frame_equal(table, expected)
```

This is failing on the last line with `DataFrame shape mismatch`.

Reply:
don't pass the ``{}``

Reply:
Thanks for the suggestion. This is what I have now:

```python
expected = pd.DataFrame(columns=['foo'], index=[1, 2, 3, greek])
```

It's still failing:

```
E       AssertionError: DataFrame are different
E
E       DataFrame shape mismatch
E       [left]:  (4, 0)
E       [right]: (4, 1)

pandas/util/testing.py:1105: AssertionError
```

What else could be wrong?